package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Serto",
   "homepage": "https://serto.id",
   "repository": "SertoID/serto-ui",
-  "version": "0.3.29",
+  "version": "0.3.30",
```

Comment:
My recent PR bumped version to `0.3.30` (not sure why it doesn't show that in the diff here), so you'll have to bump it again or else when this PR merges it won't auto-publish to NPM